setup.py

```diff
@@ -24,7 +25,6 @@ def parse_requirements(requirement_file):
         'setuptools_scm',
```

Comment:
Can remove `setuptools_scm` if not using git tags for automatic version discovery.